page-components/featured-content-module.md

```diff
@@ -39,61 +39,45 @@ Featured content modules (FCMs) function to highlight a specific piece of conten
 
 <div class="content-67 content-first">
 
-Featured content modules should include a single piece of content from a defined content type, including: Blog, News, Video, Event, and Tool.
+Featured content modules should highlight a single piece of content. That content can be almost anything, from a webpage to a blog post to a PDF to a video.
 
 FCMs appear below the title and introductory paragraph on a page, and retain that position on all screen sizes.
 
 FCMs contain a headline, body copy text, and a graphical element related to the piece of content being featured. They are a “mini story” and never simply a dominant visual.
```

Comment:
Should we add "call to action link" to this description? Maybe, "FCMs contain a headline, body copy text, call to action link if applicable, and graphical element related to the piece of content being featured."